js.dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:10-slim
+
+RUN apt-get update && apt-get install -yq libgconf-2-4 \
+    && apt-get -y install sudo \
```

Comment:
If you end up not having to use a different user, then `sudo` becomes redundant. See comment below.